python/src/otel/otel_sdk/otel-instrument

```diff
@@ -89,14 +89,9 @@ export PYTHONPATH="$LAMBDA_RUNTIME_DIR:$PYTHONPATH";
 
 # - Set the trace exporter
```

Comment:
Could we change/remove this line, would be easy to understand

Reply:
to understand the script

Reply:
done